lxd/daemon.go

```diff
@@ -317,70 +317,76 @@ func allowPermission(entityType entity.Type, entitlement auth.Entitlement, muxVa
 
 // allowProjectResourceList should be used instead of allowAuthenticated when listing resources within a project.
 // This prevents a restricted TLS client from listing resources in a project that they do not have access to.
-func allowProjectResourceList(d *Daemon, r *http.Request) response.Response {
-	requestor, err := request.GetRequestor(r.Context())
-	if err != nil {
-		return response.SmartError(err)
-	}
+func allowProjectResourceList(allowAllProjects bool) func(d *Daemon, r *http.Request) response.Response {
```

Comment:
allowAllProjects param is not documented in the doc block

Reply:
This controls whether a restricted TLS client is allowed to use `all-projects=true`

Reply:
Added to the doc block